arm-apimanagement/2016-10-10/swagger/apimanagement.json

```diff
@@ -0,0 +1,7693 @@
+          },
+          {
+            "$ref": "#/parameters/TopQueryParameter"
+          },
+          {
+            "$ref": "#/parameters/SkipQueryParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Lists a collection of user entities associated with the group.",
+            "schema": {
+              "$ref": "#/definitions/UserCollection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorBodyContract"
+            }
+          }        
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/UserContract"
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/groups/{groupId}/users/{uid}": {
+      "put": {
+        "tags": [
+          "GroupUsers"
+        ],
+        "operationId": "GroupUsers_Add",
+        "description": "Adds a user to the specified group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/GroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/UserIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "The user was successfully added to the group."
+          },
+          "204": {
+            "description": "The specified user is already a member of the specified group."
+          },
+          "405": {
+            "description": "Attempt was made to add a user to a built-in group. Built-in group membership is managed by the system.",
+            "schema": {
+              "$ref": "#/definitions/ErrorBodyContract"              
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorBodyContract"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "GroupUsers"
+        ],
+        "operationId": "GroupUsers_Remove",
```

Comment:
#fixed

---
In reply to: [106752535](https://github.com/Azure/azure-rest-api-specs/pull/1039#discussion_r106752535) [](ancestors = 106752535)